guides/source/i18n.md

```diff
@@ -872,33 +872,33 @@ So, for example, instead of the default error message `"cannot be blank"` you co
 
 * `count`, where available, can be used for pluralization if present:
 
-| validation   | with option               | message                   | interpolation |
-| ------------ | ------------------------- | ------------------------- | ------------- |
-| confirmation | -                         | :confirmation             | attribute     |
-| acceptance   | -                         | :accepted                 | -             |
-| presence     | -                         | :blank                    | -             |
-| absence      | -                         | :present                  | -             |
-| length       | :within, :in              | :too_short                | count         |
-| length       | :within, :in              | :too_long                 | count         |
-| length       | :is                       | :wrong_length             | count         |
-| length       | :minimum                  | :too_short                | count         |
-| length       | :maximum                  | :too_long                 | count         |
-| uniqueness   | -                         | :taken                    | -             |
-| format       | -                         | :invalid                  | -             |
-| inclusion    | -                         | :inclusion                | -             |
-| exclusion    | -                         | :exclusion                | -             |
-| associated   | -                         | :invalid                  | -             |
-| non-optional association | -             | :required                 | -             |
-| numericality | -                         | :not_a_number             | -             |
-| numericality | :greater_than             | :greater_than             | count         |
-| numericality | :greater_than_or_equal_to | :greater_than_or_equal_to | count         |
-| numericality | :equal_to                 | :equal_to                 | count         |
-| numericality | :less_than                | :less_than                | count         |
-| numericality | :less_than_or_equal_to    | :less_than_or_equal_to    | count         |
-| numericality | :other_than               | :other_than               | count         |
-| numericality | :only_integer             | :not_an_integer           | -             |
-| numericality | :odd                      | :odd                      | -             |
-| numericality | :even                     | :even                     | -             |
+| validation               | with option               | message                   | interpolation |
```

Comment:
Yes, I referred to the table column changes. Because every new change also needs to ensure the alignment. That's the hard part.